docs/application/native/tutorials/details/sec-privileges.md

```diff
@@ -28,242 +28,140 @@ according to their hierarchy:
+| `http://tizen.org/privilege/mediacontroller.server` | public |  | The application can send information about currently playing media to applications that are allowed to receive it, and can be controlled remotely by those applications. |
+| `http://tizen.org/privilege/mediahistory.read` | public | User history | The application can read the statistics concerning the music and videos played on the device, such as the peak times for playing music or videos. |
+| `http://tizen.org/privilege/message.read` | public | Message | The application can read text and multimedia messages, and any information related to them. |
+| `http://tizen.org/privilege/message.write` | public | Message | The application can write, send, delete, move text and multimedia messages, download multimedia messages, and change the settings and statuses of messages, such as read or unread. This may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/minicontrol.provider` | public |  | The application can show a small toolbar on the notification panel or lock screen while it is open. |
+| `http://tizen.org/privilege/network.get` | public |  | The application can retrieve network information such as the status of each network, its type, and detailed network profile information. |
+| `http://tizen.org/privilege/network.profile` | public |  | The application can add, remove, and edit network profiles. |
+| `http://tizen.org/privilege/network.set` | public |  | The application can turn Wi-Fi on and off, and connect to and disconnect from Wi-Fi and mobile networks. This may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/nfc` | public |  | The application can read and write NFC tag information, and send NFC messages to other devices. |
+| `http://tizen.org/privilege/nfc.admin` | platform |  | The application can change NFC settings, such as turning NFC on or off. |
+| `http://tizen.org/privilege/nfc.cardemulation` | public |  | The application can access smart card details, such as credit card details, and allow users to make payments via NFC. |
+| `http://tizen.org/privilege/notification` | public |  | The application can show and hide its own notifications and badges. |
+| `http://tizen.org/privilege/packagemanager.admin` | platform |  | The application can install and uninstall application packages. |
+| `http://tizen.org/privilege/packagemanager.clearcache` | public |  | The application can clear other applications' caches. |
+| `http://tizen.org/privilege/packagemanager.info` | public |  | The application can retrieve detailed application package information. |
+| `http://tizen.org/privilege/peripheralio` | platform |  | The application can communicate with peripherals using industry standard protocols and interfaces, such as GPIO, I2C, PWM, UART, and SPI. |
```

Comment:
Do you mean those 4 privileges should be listed on the separated table? Or just merge them into privilege table with no profile?

Reply:
They should be listed on the separated table.

You can refer [here](https://github.com/Samsung/tizen-docs/blob/tizen_5.0_prepare/docs/application/native/tutorials/details/sec-privileges.md#iot-native-api-privileges).

![image](https://user-images.githubusercontent.com/9958374/47640906-0f26f480-dba8-11e8-9f6e-410bc24b4879.png)

I think you can remove the "Since" info from the table.

Reply:
removed the "since" column and resolved confilct.